CloudinaryDotNet/ApiShared.Internal.cs

```diff
@@ -248,6 +248,26 @@ protected void HandleUnsignedParameters(IDictionary<string, object> parameters)
             }
         }
 
+        private static void AddCommentToUserAgent(
+                HttpHeaderValueCollection<ProductInfoHeaderValue> userAgentHeader,
+                string comment)
+        {
+            if (string.IsNullOrEmpty(comment))
+            {
+                return;
+            }
+
+            var normalizedComment = RemoveBracketsFrom(comment);
```

Comment:
Please explain the problem we are solving here. In a year nobody will remember why we remove brackets from headers.